src/pages/IstioConfigDetails/IstioConfigDetailsPage.tsx

```diff
@@ -192,8 +193,16 @@ class IstioConfigDetailsPage extends React.Component<RouteComponentProps<IstioCo
     // Hack to force redisplay of annotations after update
     // See https://github.com/securingsincity/react-ace/issues/300
     if (this.aceEditorRef.current) {
+      const editor = this.aceEditorRef.current!['editor'];
+
       // tslint:disable-next-line
-      this.aceEditorRef.current!['editor'].onChangeAnnotation();
+      editor.onChangeAnnotation();
+
+      // Fold Status field
```

Comment:
Ok, if the status is not present, this is failing in my case I guess I didn't test it with the status enabled.